src/content/packages-and-plugins/swift-package-manager/for-app-developers.md

```diff
@@ -0,0 +1,144 @@
+the project, you might get an error like this:
+
+```plaintext
+Target Integrity (Xcode): The package product 'plugin_name_ios' requires minimum platform version 14.0 for the iOS platform, but this target supports 12.0
+```
+
+To use the plugin, you'll need to increase the **Minimum Deployments** of your
+app's target.
+
+:::warning
+This raises your app's minimum OS version requirement.
+:::
+
+{% render docs/captioned-image.liquid,
+image:"development/packages-and-plugins/swift-package-manager/minimum-deployments.png",
+caption:"The target's **Minimum Deployments** setting" %}
+
+## How to add Swift Package Manager integration to a custom Xcode target
+
+Your Flutter Xcode project can have custom [Xcode targets][] to build additional
+products, like frameworks or unit tests.
+You can add Swift Package Manager integration to these custom Xcode targets
+as well.
+
+Follow the steps in
+[How to add Swift Package Manager integration to a project manually][manualIntegration].
+
+In [Step 1][manualIntegrationStep1], list item 6 use your custom target instead
+of the `Flutter` target.
+
+In [Step 2][manualIntegrationStep2], list item 6 use your custom target instead
+of the `Flutter` target.
+
+[Xcode targets]: https://developer.apple.com/documentation/xcode/configuring-a-new-target-in-your-project
+[manualIntegration]: /packages-and-plugins/swift-package-manager/for-app-developers/#how-to-add-swift-package-manager-integration-to-a-project-manually
+[manualIntegrationStep1]: /packages-and-plugins/swift-package-manager/for-app-developers/#step-1-add-fluttergeneratedpluginswiftpackage-package-dependency
+[manualIntegrationStep2]: /packages-and-plugins/swift-package-manager/for-app-developers/#step-2-add-run-prepare-flutter-framework-script-pre-action
+
+## How to remove Swift Package Manager integration
+
+The Flutter CLI migrates your project to add Swift Package Manager integration.
+This makes your project download the Swift packages that your Flutter plugins
+depend on.
+
+To undo this migration:
+
+1. [Turn off Swift Package Manager][].
+
+2. Open your app (`ios/Runner.xcworkspace` or `ios/Runner.xcworkspace`) in
+   Xcode.
+
+3. Navigate to **Package Dependencies** for the project.
+
+4. Click on the `FlutterGeneratedPluginSwiftPackage` package and then click the
+   **-** button.
+
+   {% render docs/captioned-image.liquid,
+   image:"development/packages-and-plugins/swift-package-manager/remove-generated-package.png",
+   caption:"The `FlutterGeneratedPluginSwiftPackage` to remove" %}
+
+4. Navigate to **Frameworks, Libraries, and Embedded Content** for the `Runner`
+   target.
+
+5. Click on `FlutterGeneratedPluginSwiftPackage` and then click the **-**
+   button.
+
+   {% render docs/captioned-image.liquid,
+   image:"development/packages-and-plugins/swift-package-manager/remove-generated-framework.png",
+   caption:"The `FlutterGeneratedPluginSwiftPackage` to remove" %}
+
+6. Select **Product > Scheme > Edit Scheme**.
```

Comment:
**suggestion:** Phrase as _Go to_ instead of _Select_ per [GSG](https://developers.google.com/style/ui-elements#ui-terms).

```suggestion
6. Go to **Product > Scheme > Edit Scheme**.
```

Reply:
Updated!